src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,51 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int MAX_VALUE = 10;
+    private static final int VALUE_NOT_FOUND = -1;
+    private K[] keyList;
+    private V[] valueList;
+    private int count = 0;
+
+    public StorageImpl() {
+        keyList = (K[]) new Object[MAX_VALUE];
+        valueList = (V[]) new Object[MAX_VALUE];
+    }
+
     @Override
     public void put(K key, V value) {
+        int index = getIndex(key);
+        if (index != VALUE_NOT_FOUND) {
+            valueList[index] = value;
+        } else {
+            keyList[count] = key;
+            valueList[count] = value;
+            count++;
+        }
     }
 
     @Override
     public V get(K key) {
-        return null;
+        int index = getIndex(key);
+        if (index != VALUE_NOT_FOUND) {
+            return (V) valueList[index];
+        } else {
+            return null;
+        }
```

Comment:
```suggestion
        } 
        return null;
```
we don't need else after return